pkg/controller/istiocontrolplane/pruningdetails.go

```diff
@@ -17,10 +17,9 @@ package istiocontrolplane
 import (
 	"strconv"
 
-	"istio.io/operator/pkg/apis/istio/v1alpha1"
-
 	"k8s.io/apimachinery/pkg/runtime/schema"
 
+	"istio.io/operator/pkg/apis/istio/v1alpha1"
```

Comment:
In #34 values is moved to `v1alpha2` as well, so after a rebase this should be changed

Reply:
actually this is the old API (this is Rob's controller code) so this is the correct API - for some reason goimports decided to clean up this file for me and include in the review.